### DISCUSSION TITLE
Use of Grafana Tempo for Visualization

### DISCUSSION THREAD

**Discussion by jobinjosem1** [signalfx/splunk-otel-collector#4707] [Q&A]
We intend to leverage splunk-otel-collector to forward logs from our applications running on AKS to splunk cloud. We are also interested in exploring the possibility of using the splunk-otel-collector to send traces to our Grafana Tempo instance for visualization. Does the otel exporter supports sending traces to Tempo endpoint? Any assistance on this matter would be greatly appreciated.

**Comment by chenlujjj** [comment]
Hi @jobinjosem1, yes it can export tracing data to Tempo via the otlp exporter. I've tested it.
